Projects/Server/Server.csproj

```diff
@@ -37,34 +39,10 @@
     <OutDir>..\..\Distribution</OutDir>
     <OutputPath>..\..\Distribution</OutputPath>
     <LangVersion>8.0</LangVersion>
+    <DebugSymbols>true</DebugSymbols>
+    <DefineConstants>DEBUG</DefineConstants>
+    <Optimize>true</Optimize>
   </PropertyGroup>
-  <ItemGroup>

```

Comment:
Why remove this?
```
    <Content Include="Assemblies\libuv.dll" Condition="'$(RuntimeIdentifier)'=='win-x64'">	
      <Link>libuv.dll</Link>	
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>	
    </Content>
    <Content Include="Assemblies\zlib.dll" Condition="'$(RuntimeIdentifier)'=='win-x64'">	
      <Link>zlib.dll</Link>	
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>	
    </Content>	
    <Content Include="Assemblies\rdrand.dll" Condition="'$(RuntimeIdentifier)'=='win-x64'">	
      <Link>rdrand.dll</Link>	
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>	
    </Content>
```

That is needed to move the file to the Distribution folder for Windows.